packages/@react-spectrum/actionbar/docs/ActionBar.mdx

```diff
@@ -0,0 +1,281 @@
+    </ActionBarContainer>
+  );
+}
+```
+
+### Accessibility
+The contents of the ActionBar should follow the same accessibility guidelines as ActionGroup's items, see the [ActionGroup docs](./ActionGroup.html#accessibility) for more details.
+
+### Internationalization
+To internationalize an ActionBar, the content provided to all child items should be localized. A localized string for the selection count and clear button `aria-label` will be provided
+automatically. For languages that are read right-to-left (e.g. Hebrew and Arabic), the layout of the ActionBar is automatically flipped.
+
+## Events
+Use the `onClearSelection` prop as a callback to handle when the clear button in the ActionBar is pressed. Additionally, the `onAction` prop can be used as a callback for when the user
+presses any of the buttons in the ActionBar's ActionGroup. The `key` from the pressed button will be passed into the callback.
+
+The following example clears row selection within the TableView when the clear button is pressed and alerts the user as to what
+action is being taken on the selected rows when the ActionButtons are pressed.
+
+```tsx example
+function ActionBarActions(props) {
+  // Same bar items, table columns, and table rows as example above...
+  ///- begin collapse -///
+  let columns = [
+    {name: 'Name', uid: 'name'},
+    {name: 'Type', uid: 'type'},
+    {name: 'Level', uid: 'level'}
+  ];
+  let rows = [
+    {id: 1, name: 'Charizard', type: 'Fire, Flying', level: '67'},
+    {id: 2, name: 'Blastoise', type: 'Water', level: '56'},
+    {id: 3, name: 'Venusaur', type: 'Grass, Poison', level: '83'},
+    {id: 4, name: 'Pikachu', type: 'Electric', level: '100'}
+
+  ];
+  let barItems = [
+    {key: 'edit', label: 'Edit'},
+    {key: 'copy', label: 'Copy'},
+    {key: 'delete', label: 'Delete'}
+  ];
+  let [selectedKeys, setSelectedKeys] = React.useState(new Set([2]));
+  ///- end collapse -///
+  return (
+    <ActionBarContainer height="size-5000">
+      <TableView aria-label="Table with controlled selection" selectionMode="multiple" selectedKeys={selectedKeys} onSelectionChange={setSelectedKeys} height="size-3000">
+        <TableHeader columns={columns}>
+          {column => (
+            <Column key={column.uid} align={column.uid === 'level' ? 'end' : 'start'}>
+              {column.name}
+            </Column>
+          )}
+        </TableHeader>
+        <TableBody items={rows}>
+          {item => (
+            <Row>
+              {columnKey => <Cell>{item[columnKey]}</Cell>}
+            </Row>
+          )}
+        </TableBody>
+      </TableView>
+      {/*- begin highlight -*/}
+      <ActionBar
+        items={barItems}
+        isEmphasized
+        selectedItemCount={selectedKeys === 'all' ? selectedKeys : selectedKeys.size}
+        onClearSelection={() => {
+          setSelectedKeys(new Set());
+        }}
+        onAction={key => alert(`Performing ${key} action on ${selectedKeys.size} rows.`)}>
```

Comment:
Maybe highlight these two event handlers specifically to make it more specific to the section this example is under

Reply:
Same as above